packages/rn-tester/package.json

```diff
@@ -25,7 +25,8 @@
   },
   "peerDependencies": {
     "react": "18.2.0",
-    "react-native": "*"
+    "react-native": "*",
+    "@react-native/virtualized-lists":"*"
```

Comment:
See the other comment about importing through React Native, which would also let us remove the explicit dependency on `@react-native/virtualized-lists`.

(Separately none of these should be peerDependencies since rn-tester is a package for an app, that nothing else includes. I will fix that later...)

Reply:
Fixed